apps/dotcom/client/src/tla/components/TlaButton/TlaButton.tsx

```diff
@@ -38,13 +40,16 @@ export const TlaButton = forwardRef<
 				className
 			)}
 		>
-			{icon && <TlaIcon icon={icon} />}
-			{children && <span>{children}</span>}
-			{iconRight && <TlaIcon icon={iconRight} className={styles.iconRight} />}
-			{isLoading && (
+			{isLoading ? (
```

Comment:
Looks like this logic was never used and was bulled. The spinner was show on top of other content.